scripts/console

```diff
@@ -4,7 +4,7 @@ set -e
 function usage() {
```

Comment:
We have an updated version of the `console` STRTA.

https://github.com/azavea/american-water-tank-magic/blob/develop/scripts/console